include/turtle/log.hpp

```diff
@@ -51,16 +50,17 @@ namespace detail
             s << (it == begin ? "" : ",") << mock::format( *it );
         s << ')';
     }
+    template<typename T>
+    struct is_callable_impl: std::false_type
+    {};
+    template<typename R, typename... Args>
+    struct is_callable_impl<R(Args...)>: std::true_type
+    {};
+    template<typename T>
+    struct is_callable: is_callable_impl< std::remove_cv_t<T> >
+    {};
```

Comment:
https://www.boost.org/doc/libs/master/libs/callable_traits/doc/html/callable_traits/reference.html#callable_traits.reference.ref_is_invocable ?

Reply:
Not really. I tried to translate what was there before and this whole thing isn't really good anyway: Besides a plain function there isn't a Type "callable" (this trait only takes a single parameter!)
Actually you would need to check if you can invoke something with specific parameters as `is_invocable` does. But we don't have those.

So a better name would be e.g. `is_function`.